cinema/urls.py

```diff
@@ -8,16 +10,39 @@
     MovieViewSet,
     MovieSessionViewSet,
     OrderViewSet,
+    # MovieDetailView,
+    # MovieSessionDetailView,
+)
+
+
+movie_list = MovieViewSet.as_view({"get": "list", "post": "create"})
+movie_detail = MovieViewSet.as_view({"get": "retrieve"})
+movie_session_list = MovieSessionViewSet.as_view(
+    {"get": "list", "post": "create"}
+)
+movie_session_detail = MovieSessionViewSet.as_view(
+    {
+        "get": "retrieve",
+        "put": "update",
+        "patch": "partial_update",
+        "delete": "destroy",
+    }
 )
 
-router = routers.DefaultRouter()
-router.register("genres", GenreViewSet)
-router.register("actors", ActorViewSet)
-router.register("cinema_halls", CinemaHallViewSet)
-router.register("movies", MovieViewSet)
-router.register("movie_sessions", MovieSessionViewSet)
-router.register("orders", OrderViewSet)
+urlpatterns = [
+    path("movies/", movie_list, name="movie-list"),
+    path("movies/<int:pk>/", movie_detail, name="movie-detail"),
+    path("actors/", ActorViewSet.as_view(), name="actor-list"),
+    path("cinema_hall/", CinemaHallViewSet.as_view(), name="cinemahall-list"),
+    path("genres/", GenreViewSet.as_view(), name="genre-list"),
+    path("movie_sessions/", movie_session_list, name="moviesession-list"),
+    path(
+        "movie_sessions/<int:pk>/",
+        movie_session_detail,
+        name="moviesession-detail",
+    ),
+    path("orders/", OrderViewSet.as_view(), name="order-list"),
```

Comment:
The `OrderViewSet` is included as a path, but ensure that it only allows list and create actions as per the task requirements. If not, you might need to adjust the viewset or the URL configuration accordingly.